File: app/src/androidTest/java/com/example/newsapp/datamodel/DataModelTestBuilder.java
package com.example.newsapp.datamodel;

import data.datamodels.Articles;

public class DataModelTestBuilder {

    public static Articles createArticle(String author,
                                         String title,
                                         String description,
                                         String url,
                                         String urlToImage,
                                         String publishedAt,
                                         String content){

        Articles articles = new Articles();

        articles.setAuthor(author);
        articles.setTitle(title);
        articles.setDescription(description);
        articles.setUrl(url);
        articles.setUrlToImage(urlToImage);
        articles.setPublishedAt(publishedAt);
        articles.setContent(content);
        return  articles;
    }
}

File: app/src/androidTest/java/com/example/newsapp/datamodel/DataModelTestBuilder.kt
package com.example.newsapp.datamodel

import com.example.newsapp.data.datamodels.Articles

object DataModelTestBuilder {

    fun createArticle(author: String,
                      title: String,
                      description: String,
                      url: String,
                      urlToImage: String,
                      publishedAt: String,
                      content: String): Articles {

        val articles = Articles()

        articles.author = author
        articles.title = title
        articles.description = description
        articles.url = url
        articles.urlToImage = urlToImage
        articles.publishedAt = publishedAt
        articles.content = content
        return articles
    }
}
